Packages/Netherlands3D/SelectionTools/Runtime/Scripts/PolygonVisualisationInput.cs

```diff
@@ -0,0 +1,65 @@
+/*
+*  Copyright (C) X Gemeente
+*                X Amsterdam
+*                X Economic Services Departments
+*
+*  Licensed under the EUPL, Version 1.2 or later (the "License");
+*  You may not use this work except in compliance with the License.
+*  You may obtain a copy of the License at:
+*
+*    https://github.com/Amsterdam/3DAmsterdam/blob/master/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" basis,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+*  implied. See the License for the specific language governing
+*  permissions and limitations under the License.
+*/
+using Netherlands3D.Events;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Netherlands3D.SelectionTools
+{
+    [RequireComponent(typeof(PolygonVisualiser))]
+    public class PolygonVisualisationInput : MonoBehaviour
+    {
+        private PolygonVisualiser polygonVisualiser;
+
+        [Header("Listen to events:")]
+        [SerializeField] private Vector3ListsEvent drawPolygonEvent;
+        [SerializeField] private Vector3ListEvent drawSinglePolygonEvent;
+        [SerializeField] private StringEvent setDrawingObjectName;
+        [SerializeField] private FloatEvent setExtrusionHeightEvent;
+        //[SerializeField] private GameObjectEvent polyParentEvent;
```

Comment:
Gecommente code had weggehaald kunnen worden; kan in een volgende opschoon gefixt worden

Reply:
Deze had ik bewust nog gelaten omdat ik niet zeker wist of dit gebruikt werd, volgens mij niet daarom heb ik het gecomment